wallet-test/src/test/scala/org/bitcoins/wallet/models/AccountDAOTest.scala

```diff
@@ -4,9 +4,9 @@ import org.bitcoins.core.api.wallet.db.AccountDb
 import org.bitcoins.testkit.Implicits._
 import org.bitcoins.testkit.core.gen.CryptoGenerators
 import org.bitcoins.testkit.fixtures.WalletDAOFixture
-import org.bitcoins.testkit.wallet.{BitcoinSWalletTest, WalletTestUtil}
+import org.bitcoins.testkit.wallet.WalletTestUtil
 
-class AccountDAOTest extends BitcoinSWalletTest with WalletDAOFixture {
+class AccountDAOTest extends WalletDAOFixture {
```

Comment:
Why did we need to remove `extends BitcoinSWalletTest` on all of these?

Reply:
It was putting 2 different `WalletAppConfig`s in context here, 1 from `BitcoinSWalletTest` and ` from `WalletDAOFixture`.